core/src/test/scala/unit/kafka/security/auth/ZkAuthorizationTest.scala

```diff
@@ -55,9 +56,11 @@ class ZkAuthorizationTest extends ZooKeeperTestHarness with Logging{
   @Test
   def testIsZkSecurityEnabled() {
     assertTrue(JaasUtils.isZkSecurityEnabled())
+    Configuration.setConfiguration(null)
```

Comment:
@rajinisivaram I think we were loading the configuration before as well in `JaasUtils.isZkSecurityEnabled()`, no? I'm interested in this because this issue is quite annoying so I'd like to track it every time I see, it is not necessarily a blocker for this PR.